sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerConfig.java

```diff
@@ -57,6 +60,8 @@
 
   public abstract @Nullable ValueProvider<Duration> getMaxCumulativeBackoff();
 
+  public abstract @Nullable ValueProvider<RpcPriority> getRpcPriority();
```

Comment:
This does not have to be a ValueProvider argument, especially since it's not really exposed to users (they only use `withLowPriority` or `withHighPriority` to set this). You can make it directly `RpcPriority` type

Reply:
It seems that we have a default value for it, so it doesn't have to be `@Nullable` either, right?

Reply:
Done